tensorboard/plugins/projector/vz_projector/util.ts

```diff
@@ -248,4 +250,9 @@ export function hasWebGLSupport(): boolean {
   }
 }
 
+/** Checks to see if input is undefined or null. */
+export function exists(x: any): boolean {
```

Comment:
IMO this doesn’t merit its own function. The pattern `x != null` is
sufficiently common in JavaScript/TypeScript (192 uses in our codebase)
that when reading `util.exists` I assume that this function must be
doing something _else_ and have to look it up to see that in fact it is
doing the same thing.

Although <http://go/tsstyle> (Google-internal) bans `!=` in general, the
particular case of `!= null` is explicitly permitted.

Reply:
Fine with me! Though I do have an irrational fear of coercion...